### PR TITLE
GH-448 Add data access page

### PR DESCRIPTION
**Pull request by sgratiy**
# Addresses: 
issue #448 
# Type of Fix:
Documentation Change

# Tasks:
- [x] Add data access page
- [x] Add comment in the gallery of examples to refer to docs (data access page) for downloading data
- [x] Create utility for accessing file manifest and experiment metadata. This are temporally committed as part of the repo until this data is publicly available through brain-map.org

### CODE REVIEW

File: docs/download_data.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+.. code-block:: bash
+
+    dandi download --output-dir <DIRECTORY> <URL>
+
+where <DIRECTORY> is the existing directory on your file system
+and <URL> is the url path of a file or archive.
+
+The paths to individual data files withing the archive are listed in the file manifest
+(need links) under the ``arhive_uri`` column.
```
Comment (NileGraddis) [inline docs/download_data.rst:31]: "archive_uri" typo

Comment (NileGraddis) [reply docs/download_data.rst:31]: also didn't we get these file manifests last week?

**Comment by NileGraddis** [comment]
It seems like we really need to get our hands on these manifest files in order to provide end-to-end download instructions.

Once we do, it might be nice to include a short Pandas/Requests based snippet that downloads a file based on some criteria and makes an EphysDataSet from it.

File: docs/download_data.rst (hunk 1 above)
Comment (sgratiy) [reply docs/download_data.rst:31]: per discussion with @wbwakeman, the manifests will be posted on a "Explore" page on brain-map.org. When this page is live we should be able to link to it.

**Comment by sgratiy** [comment]
>Once we do, it might be nice to include a short Pandas/Requests/dandi based snippet that downloads a file based on some criteria and makes an EphysDataSet from it.
I agree that this would be handy. I would suggest though to ask scientists for the use cases before implementing it.

**Comment by sgratiy** [comment]
@NileGraddis I have added the utility and committed the manifest/metdata. please take a look again, try it out, and merge if acceptable.

**Comment by wbwakeman** [comment]
Updated (but still not yet final due to #428 ) versions of the metadata and manifest files are available at https://alleninstitute-my.sharepoint.com/:f:/g/personal/waynew_alleninstitute_org/El0YAZ94AYNHpvODWrTwFloBdV13h-kH9DSKYOMUvEg_CA?e=eoHFAv .  Those should be used instead.

Typo: chnage 

Grammar: "satisfying a particular experimental conditions." should be "satisfying particular experimental conditions."

**Comment by NileGraddis** [comment]
OK, I've 
1. made the updates/typo fixes that @wbwakeman suggested.
2. swapped "organism" -> "dataset" in the ARCHIVE_INFO (in case we release future datasets that distinct from these but use the same organisms).
3. added csvs to the MANIFEST.in so that users who pip install IPFX can use them.

**Pull request merged**